nh_ews/models/nh_clinical_frequencies_ews.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from openerp import api
+from openerp.models import AbstractModel
+
+
+class NhClinicalFrequenciesEws(AbstractModel):
+    """
+    This model retrieves different frequencies that may have been configured
+    for the system such as `low_risk` (the frequency of an obs for a low risk
+    patient) and `obs_restart` (the frequency of an obs created after a
+    patient has just started having observations performed on them again since
+    coming off of 'obs stop' status).
+
+    Currently the methods all retrieve configuration from the database by
+    calling `ir.config_parameter.get_param()`. This is an Odoo model that acts
+    as an interface to a simple key-value store accessible only by the
+    superuser.
+
+    All the methods call `get_param` on this model which in turn calls the
+    `ir.config_parameter` version.
+    """
+    _name = 'nh.clinical.frequencies.ews'
+
+    def get_minimum_frequency(self, clinical_risk):
+        """
+        Get the parameter which specifies the minimum frequency that can
+        safely by set for a particular risk. The parameter must follow a
+        particular naming convention that the method assumes.
+        :param clinical_risk:
```

Comment:
What do the params represent? Also need line between params and description

Reply:
I'm a bit confused what you mean. Doesn't this explain what the parameter represents?
> ...which specifies the minimum frequency that can safely by set for a particular risk.